src/app/shared/input-errors/input-errors.component.html

```diff
@@ -0,0 +1,6 @@
+<span class="custom-error">
```

Comment:
Hey, @patrickf949 , this component isn't doing much here, 
what do you think of having a `p` tag on the HTML template you are using this and conditionally render it incase of an error

Reply:
Yes @Tevinthuku, that makes sense but, this is simply a reusable component that anyone can make use of without having to style